src/test_scenes.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(unused)]
```

Comment:
This file shouldn’t be part of the library build. Should be moved to an example, along with the min.rs

Reply:
I agree, but this is out of the scope of this PR.